src/services/users.service.ts

```diff
@@ -52,6 +52,17 @@ class UserService {
     const deleteUserData = await this.users.delete({ where: { id: userId }, select: { email: true, id: true } });
     return deleteUserData;
   }
+
+  public async manageGroups(userId: number, data: any) {
```

Comment:
Missing return type declaration

Reply:
As well as "data" parameter type declaration, over all this function is a bit unclear especially the "groups" variable declaration which uses the users client (it also requires linting).